pkg/controller/hyperconverged/hyperconverged_controller.go

```diff
@@ -106,17 +107,29 @@ func (r *ReconcileHyperConverged) Reconcile(request reconcile.Request) (reconcil
 		return reconcile.Result{}, err
 	}
 
-        if instance.Spec.KubeVirtImagePullPolicy != "" {
-           reqLogger := log.WithValues("imagePullPolicy", instance.Spec.KubeVirtImagePullPolicy)
-           reqLogger.Info("HCO CR contains KubeVirt Image Pull Policy")
-           KubeVirtImagePullPolicy = instance.Spec.KubeVirtImagePullPolicy
-        }
+	if instance.Spec.KubeVirtImagePullPolicy != "" {
```

Comment:
I'm wondering if we can improve the abstraction here for validating input.   We are going to need to add code like this for every configuration we support, so this func will fill up fast.

Reply:
> 
> 
> I'm wondering if we can improve the abstraction here for validating input. We are going to need to add code like this for every configuration we support, so this func will fill up fast.

It's surely something to think about for the future, i.e. creating some sort of parameter validation framework, but this out of the scope of the current task IMHO.